app/models/users/user.py

```
@@ -12,11 +52,10 @@
 from app.models.base import Base
 
 
-class UserType(enum.Enum):
+class UserType(str, Enum):
```

Ruff (0.11.9)

Undefined name `Enum`

(F821)